below/tc/src/lib.rs

```diff
@@ -0,0 +1,111 @@
+mod errors;
+mod types;
+
+#[cfg(test)]
+mod test;
+
+use std::collections::BTreeMap;
+
+use netlink_packet_core::{NetlinkHeader, NetlinkMessage, NetlinkPayload, NLM_F_DUMP, NLM_F_REQUEST};
+use netlink_packet_route::RouteNetlinkMessage;
+use netlink_packet_route::tc::TcMessage;
+use netlink_sys::constants::NETLINK_ROUTE;
+use netlink_sys::{Socket, SocketAddr};
+use nix::net::if_;
+
+pub use errors::*;
+pub use types::*;
```

Comment:
explicit imports usually better. so that during code review it's easier to spot where types come from